src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/identifiers/PNCIdentifierTest.kt

```diff
@@ -55,41 +52,22 @@ class PNCIdentifierTest {
     strings = ["01", "012", "0123", "01234", "012345", "0123567", "012345678", "0123456789", "01234567890", "01234567890123"],
   )
   fun `should return invalid when PNC id is not the correct length`(pncId: String) {
-    // When
     val pncIdentifier = PNCIdentifier.from(pncId)
-
-    // Then
-    assertThat(pncIdentifier.valid).isFalse()
+    assertThat(pncIdentifier.pncId).isEqualTo("")
   }
 
   @ParameterizedTest
   @ValueSource(strings = ["TOTALLYINVALID", "1X23/1234567A", "1923[1234567A", "1923/1Z34567A", "1923/1234567AA"])
   fun `should return invalid when PNC id is incorrectly formatted`(pncId: String) {
-    // When
     val pncIdentifier = PNCIdentifier.from(pncId)
-
-    // Then
-    assertThat(pncIdentifier.valid).isFalse()
-  }
-
-  @ParameterizedTest
-  @ValueSource(strings = ["2008/0056560Z", "20030011985X", "20120052494Q", "20230583843L", "2001/0171310W", "2011/0275516Q", "2008/0056560Z", "2003/0062845E", "1981/0154257C"])
```

Comment:
This is covered by `should NOT convert PNCs already in canonical form` and `should convert short form PNC ids to canonical form`